php/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "custom-auth-examples",
+    "version": "0.0.0",
+    "description": "This express example will get you up and running using Ionic Auth with your own server backend in nodejs.",
```

Comment:
Initial reaction: why? I just copied it from the express examp... OH! Yes I will.